similarity/manage.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import print_function
+from flask.cli import FlaskGroup
+import click
+
+import webserver
+import metrics
+from operations import HybridMetric
+import db
+
+from sqlalchemy import text
+
+PROCESS_BATCH_SIZE = 10000
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app_flaskgroup)
+
+
+@cli.command()
+def init_similarity():
+    click.echo('Copying data')
+    with db.engine.begin() as connection:
+        query = text("""
```

Comment:
we shouldn’t have sql in a management command. Can you move it (and other stuff in this file) into `db/similarity.py`?